data-sources/dework/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "dework",
+  "version": "1.0.0",
+  "description": "Dework Datasets",
+  "main": "dist/generate-datasets.js",
+  "scripts": {
+    "lint": "eslint . --ext .ts",
+    "build": "tsc",
+    "start": "node dist/generate-datasets.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Nation3",
+  "license": "GPL-3.0-or-later",
+  "devDependencies": {
+    "@types/node": "^18.11.9",
```

Comment:
you need to add web3 as a dependency

Reply:
@pythonpete32 Things seem to be working even though `web3` has not been added to `package.json`?  Or is something failing/missing on your end?